railties/lib/rails/generators/rails/plugin/templates/Gemfile.tt

```diff
@@ -1,3 +1,4 @@
+<% windows_platforms = Gem.rubygems_version >= Gem::Version.new('3.3.22') ? 'windows' : 'mswin mswin64 mingw x64_mingw' -%>
```

Comment:
Let's push this to the Ruby class that render those templates.

Reply:
Done.